src/fragmentarium/ui/display/Display.sass

```diff
@@ -14,23 +14,39 @@
       content: "⸣"
 
 @mixin sumerian
-  &--SUMERIAN,
+  &--SUMERIAN
     @include normal-font
     letter-spacing: 0.1em
 
 @mixin emesal
-  &--EMESAL,
+  &--EMESAL
     @include normal-font
     color: pink
     letter-spacing: 0.1em
 
+
 .Transliteration
   list-style: none
   padding: 0
 
   &__ControlLine
     color: darkgrey
 
+  &__RulingDollarLine
+    margin-bottom: 0.5em
+
+  &__ruling
+    border-top: 2px solid black
+    height: 1px
+    margin: 0
+
+    &--double
+      border-bottom: 2px solid black
+      padding: 0.375px
+    &--triple
+      border-bottom: 6px double black
+      padding: 0.75px
```

Comment:
Could use only `height` (or only `padding` on a single side if there are issues with height) now it is hard to figure out the exact space between the borders.

Reply:
yeah height will do it !